package.json

```diff
@@ -31,6 +31,7 @@
     "core-js": "^2.5.4",
     "jsplumb": "~2.10.0",
     "ngx-bootstrap": "~3.3.0",
+    "ngx-toastr": "^10.0.4",
     "patternfly-ng": "^4.5.1",
```

Comment:
By replacing `NotificationComponent`, `patternfly-ng` is no longer used in the project. Could you please change this dependency to be just [patternfly](https://www.npmjs.com/package/patternfly) and not `patternfly-ng`? You'll also need to remove the line `"node_modules/patternfly-ng/dist/css/patternfly-ng.min.css",` in `angular.json`.